.github/workflows/dev.yml

```diff
@@ -70,6 +70,8 @@ jobs:
   wasp:
     name: No JIT
```

Comment:
I changed this Ubuntu job to use an explicit CMake version, and ninja. It's still using no-JIT and clang.